src/java.base/share/classes/java/time/chrono/ChronoLocalDate.java

```diff
@@ -700,7 +700,10 @@ default long toEpochDay() {
      * This default implementation performs the comparison defined above.
      *
      * @param other  the other date to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value, that is less than zero if this is before {@code other},
```

Comment:
This text is incorrect. It does not match the description above re chronologies.